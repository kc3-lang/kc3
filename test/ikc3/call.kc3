quote test()
quote test(1)
quote test(1, 2)
quote test(1, 2, 3)
quote Test.test()
quote Test.test(1)
quote Test.test(1, 2)
quote Test.test(1, 2, 3)
quote first([1, 2])
first([1, 2])
quote KC3.first([2, 3])
KC3.first([2, 3])
quote (1 + 2)
(1 + 2)
