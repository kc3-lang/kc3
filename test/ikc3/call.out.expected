test()
test((U8) 1)
test((U8) 1, (U8) 2)
test((U8) 1, (U8) 2, (U8) 3)
Test.test()
Test.test((U8) 1)
Test.test((U8) 1, (U8) 2)
Test.test((U8) 1, (U8) 2, (U8) 3)
first([(U8) 1, (U8) 2])
(U8) 1
first([(U8) 2, (U8) 3])
(U8) 2
((U8) 1 + (U8) 2)
(U8) 3
